django/views/generic/dates.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 
 from django.conf import settings
+from django.contrib.admin.utils import get_fields_from_path
```

Comment:
Hi @timgraham, I searched on the project but I couldn't find any helper function that I can use to get those fields (there are some functions under `django.db.models.sql.query.py` and `django.db.models.query.py`) so I added these lines: https://github.com/django/django/pull/8307/commits/2f7aedc832f44dc8d5e5bfe7953a3ecdaebb2144. Also, I don't think that it would be good idea to extend `Meta` options class because those fields are accessible as nested attributes of its  relation fields. 

Please, let me know your opinion.